### DISCUSSION TITLE
Normalize owners and visits resources APIs

### DISCUSSION THREAD

**Discussion by arey** [spring-petclinic/spring-petclinic-rest#78] [General]
The issue #35 has introduced some changes to the REST API, particularly due to the removal of the bi-directional mapping breaking change.
Some resources have now two paths:
```
POST /petclinic/api/owners/{ownerId}/pets/{petId}/visits => implementation done
POST /petclinic/api/visits => not working
```
```
POST /petclinic/api/owners/{ownerId}/pets => implementation done
POST /petclinic/api/owners => not working
```
This discussion aims to decide which one should remains.
Our choice will impact the https://github.com/spring-petclinic/spring-petclinic-angular

@alexandre-touret your ideas are welcome.

**Comment by alexandre-touret** [comment]
Hi 
Unfortunately, it's always tricky to deal with this kind of issue.
I would say it depends of the functional requirements. Does the angular app need both paths ?
